pkg/reconciler/service/install.go

```diff
@@ -11,7 +11,12 @@ import (
 )
 
 type Install struct {
-	logger *zap.SugaredLogger
```

Comment:
as it would lead to nil-pointers if the logger is not passed during initialisation, it's maybe better use a newInstall() fct and make the logger again private